app/models/lookup_value.rb

```diff
@@ -45,6 +45,7 @@ def name
   end
 
   def value_before_type_cast
+    return read_attribute(:value) if errors.present? && errors[:value].present?
```

Comment:
Same thing as on lookup_key, under what circumstances checking `errors.present?` is necessary?